LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Eyal Eizenberg
+Copyright (c) 2015 Eyal Eizenberg,
+Copyright (c) 2016 Joel McCormick
```

Comment:
Your code was a nice example but wasn't really usable outside of the one use case. I've changed over 25% of the file to make it useful more broadly so I think this is reasonable.